src/nemo_run/config.py

```diff
@@ -259,6 +260,11 @@ def __init__(
 
         super().__init__(fn_or_cls, *args, **new_kwargs)
 
+    @classmethod
+    def from_cli(cls, fn_or_cls: Union[fdl.Buildable[_T], TypeOrCallableProducingT[_T]], *args: str) -> Config[_T]:
+        from nemo_run.core.cli_parser import parse_cli_args
```

Comment:
Can we remove this cyclic import? We can make this a function in the `cli_parser.py` file and just pass the `cls` directly. Not sure if adding this as a `classmethod` provides any advantage.

Reply:
Also this method is duplicated for `Config` and `Partial` so it would be better to just have one `cli_parser.from_cli` function that avoids the cyclical imports.